modular-docs-manual/content/topics/module_a-simple-example-of-a-modular-guide.adoc

```diff
@@ -0,0 +1,18 @@
+[id='a-simple-example-of-a-modular-guide']
+= A Simple Example of a Modular Guide
+
+Download and unzip link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/simple-mod-guide-repo.zip[an example modular guide]. The `simple-mod-guide-repo` directory included in the ZIP archive contains a very simple example of a modular guide with assemblies and modules. This section explains the contents of the directory.
+
+NOTE: To build the example guide, change into the `simple-mod-guide-repo` directory, issue the *asciidoctor master.adoc* command, and open the resulting `master.html` file in your browser.
+
```

Comment:
To explain the directory structure, you might want to show the docs/ tree in a code block and use comments and/or callouts to describe what each dir/file does.

Reply:
Thanks, Ben, I like this idea! This is what I did: https://github.com/redhat-documentation/modular-docs/pull/43/commits/1ceefe7cf57b56605a761f32ef94139bfc69fd6b  I kept the comments in the code block short and followed up with a few more detailed notes.

Let me know if you think of a better solution.

Reply:
Ack - I took a quick peek and it looks good to me!